connectors/pony_connector_sink/active_workers.pony

```diff
@@ -0,0 +1,38 @@
+use "collections"
```

Comment:
If you can import `wallaroo/core/common` then you can use `WorkerName` instead of `String` where a worker name is expected.

Reply:
`wallaroo/core/common` depends on the `wallaroo-logging` library which requires a separate build step. There's a `WorkerName` in the `wallaroo_labs/connector_wire_messages` package, which I'm already using elsewhere and which doesn't rely on `wallaroo-logging`, so I'm going to use that here unless you object.

Reply:
No objection.